### ISSUE TITLE
Exclude ramp up duration for time profiling

### ISSUE THREAD

**Issue by emily-ygz** [TPC-Council/HammerDB#233]
HammerDB v4.1 supports time profiling for all virtual users but the ramp up time is included which is not a steady state, it would be better if we don't count it.

**Comment by sm-shaw** [comment]
Yes, this is a good suggestion. The plan will be to release the current xtprof as part of v4.1 and then consider this and other potential enhancements for v4.2. 

From a practical point of view, the virtual users capturing the timings don't know as such the rampup and duration times only the monitor virtual user knows. However all the calculations are done by the monitor virtual user, so it should be possible to discard the datapoints from the rampup time before the calculations are done. 

It is also possible to output all datapoints, however this will be a lot of data. 

Other areas where time profiling enhancements could be made are in #224 #145 refering to output in JSON format.

**Comment by sm-shaw** [comment]
As additional thought it should be optional whether to discard data points or not. at 4.1 there is one option in the generic.xml file that enables creating a unique logfile or not. There is the potential with time profiling to add an additional GUI/CLI option to configure time profiling.

**Comment by emily-ygz** [comment]
These 2 are very good ideas. 
1. Optional to create a unique logfile for time profiling data
2. add GUI/CLI option, GUI would be great to visualize the latency
